Extension/src/Debugger/configurationProvider.ts

```diff
@@ -584,7 +584,11 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
 
     private showErrorIfClNotAvailable(_configurationLabel: string): boolean {
         if (!process.env.DevEnvDir || process.env.DevEnvDir.length === 0) {
-            void vscode.window.showErrorMessage(localize("cl.exe.not.available", "{0} build and debug is only usable when VS Code is run from the Developer Command Prompt for VS.", "cl.exe"));
+            void vscode.window.showErrorMessage(localize({
+                key: "cl.exe.not.available",
+                comment: ["{0} is a command option in a menu. {1} is the product name \"Developer Command Prompt for VS\"."]
+            },
+                "{0} is only usable when VS Code is run from the {1}.", `cl.exe ${this.buildAndDebugActiveFileStr()}`, "Developer Command Prompt for VS"));
```

Comment:
Since it won't indent the lines above due to the placement of the `{`, you should move this line right after the comma or move the `{` to a new line (I saw some cases in our code that do that).

```ts
            void vscode.window.showErrorMessage(localize({
                key: "cl.exe.not.available",
                comment: ["{0} is a command option in a menu. {1} is the product name \"Developer Command Prompt for VS\"."]
            }, "{0} is only usable when VS Code is run from the {1}.", `cl.exe ${this.buildAndDebugActiveFileStr()}`, "Developer Command Prompt for VS"));
```

or 

```ts
            void vscode.window.showErrorMessage(localize(
                {
                    key: "cl.exe.not.available",
                    comment: ["{0} is a command option in a menu. {1} is the product name \"Developer Command Prompt for VS\"."]
                },
                "{0} is only usable when VS Code is run from the {1}.", `cl.exe ${this.buildAndDebugActiveFileStr()}`, "Developer Command Prompt for VS"));
```

Reply:
This is better than turning off linting, I think.

Reply:
@bobbrow I disagree. I think we should just rely on our TypeScript formatter instead of trying to manually adjust the formatting.

The formatting is from

settings.json
```cpp
    "[typescript]": {
        "editor.tabSize": 4,
        "editor.defaultFormatter": "vscode.typescript-language-features",
```

Reply:
Either way, you should pick one of the two options I presented. The way it's currently formatted looks bad.

Reply:
Oh, well, it looks like I can adjust the formatting and it still works, but I think we should still disabling the eslint check anyway, since it has been a problem in the past.

Reply:
I don't have a strong opinion on removing the eslint check if format on save is enabled.

Reply:
...but the problem is that some people might disable formatOnSave and now you don't have automatic verification of the formatting in the PR anymore. If you don't think that's going to be a problem, we can remove the eslint check.

Reply:
The settings.json is checked in with the formatOnSave enabled for TypeScript, so if they change it locally it's their fault and reverting to the checked in settings.json would fix the issue. The cpptools-side has the same behavior -- there's no linter for formatting. Linting for formatting is fine with me, but it would need to use the same formatter otherwise there's a conflict and the eslint formatter is deprecated/disabled.